controller/apps.go

```diff
@@ -78,7 +78,7 @@ func (c *AppsController) getSpaceNameFromSpaceID(ctx context.Context, spaceID uu
 
 	osioSpace, err := osioclient.GetSpaceByID(ctx, spaceID)
 	if err != nil {
-		return nil, errs.WithStack(err)
+		return nil, errs.Wrapf(err, "unable to connvert space UUID %s to space name", spaceID.String())
```

Comment:
typo: connvert -> convert

Reply:
no need to explicitly call `.String()`. It is the `Stringer` interface that is automatically called by `Sprintf` and derivatives when it converts it to an `%s`.

Reply:
fixed spelling mistake (and another I found)
I've gotten rid of .String() where I can.

Reply:
ok